website/content/v1.0/introduction/theila.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Theila UI for Talos
+weight: 35
+description: "An intro to Theila - a UI for Talos clusters."
+---
+
+Once you have a Talos cluster running, you may find it easier to get insights on your cluster(s) using a visual user interface rather than the `talsctl` CLI.
+For this, Sidero Labs provides [Theila](https://github.com/siderolabs/theila), a simple, single-binary web-based visual user interface for Talos clusters.
+
+## Prerequisites
+
+You should have a Talos cluster up & running, and the `talosconfig` file for Theila to access it.
+
+## Installation
+
+Theila is published as a single static binary compiled for various platforms and architectures, as well as a container image.
+
+### Binary
+
+You can download the correct binary for your system from the [releases](https://github.com/siderolabs/theila/releases) page, or use the following commands in your terminal.
+
+```bash
+curl -Lo /usr/local/bin/theila https://github.com/siderolabs/thelia/releases/download/{{< thelia_release >}}/thelia-$(uname -s | tr "[:upper:]" "[:lower:]")-amd64
+chmod +x /usr/local/bin/talosctl
+```
+
+## Use
+
+Once installed, you can run Theila by simply running it.
+
+<!-- markdownlint-disable MD001 -->
+<!-- markdownlint-disable MD022 -->
+<!-- markdownlint-disable MD025 -->
+
+{{< tabpane lang="bash" >}}
```

Comment:
doesn't *```* works here?

Reply:
It's a tabbed pane, rather than a simple code block:
![image](https://user-images.githubusercontent.com/113170/175975849-8d23c729-4623-450b-a3b4-82c1f930cdee.png)
![image](https://user-images.githubusercontent.com/113170/175975790-c57846c9-7f38-42cd-80af-91e8c0f351d0.png)

Reply:
oh i see, nice